app/_assets/javascripts/app.js

```diff
@@ -336,16 +336,113 @@ jQuery(function () {
     ); // Adjust scroll speed
   }
 
+
+// Plugin filter all / only available
+var pluginsAllOrOnly = "all";
+var showPlusWithEnterprise = false;
+function showPluginBanner(target){
+  var hubCards = $(".hub-cards");
+  if (!hubCards.length){
+    return;
+  }
+
+  var allOnlyToggle = $('#plugin-banner');
+  var eePlusBanner = $('#ee-plus-banner');
+
+  var allowedFilters = ["plus", "ee-compat"]
+  if (allowedFilters.indexOf(target) === -1){
+    pluginsAllOrOnly = "all";
+    allOnlyToggle.hide();
+    return;
+  }
+
+  allOnlyToggle.show();
+
+  if (!allOnlyToggle.length){
+    allOnlyToggle = $('<div id="plugin-banner" class="content" style="padding:0"><blockquote class="note"></blockquote></div>');
+    hubCards.prepend(allOnlyToggle)
+  }
+
+  if (!eePlusBanner.length){
+    eePlusBanner = $('<div id="ee-plus-banner" class="content" style="padding:0"><blockquote class="important">If you run Kong Gateway on your own infrastructure, you also have access to all <strong>Plus</strong> plugins. <u id="show-plus-ee">Click Here</u> to show them</blockquote></div>');
```

Comment:
```suggestion
    eePlusBanner = $('<div id="ee-plus-banner" class="content" style="padding:0"><blockquote class="important">If you run Kong Gateway on your own infrastructure, you also have access to all <strong>Plus</strong> plugins. <u id="show-plus-ee">Show Plus plugins?</u> </blockquote></div>');
```
Minor suggestion for consistency, and we should avoid using "click here".